cpp/src/ray/test/cluster/cluster_mode_test.cc

```diff
@@ -228,6 +228,53 @@ TEST(RayClusterModeTest, ExceptionTest) {
   EXPECT_THROW(object1.Get(), ray::internal::RayTaskException);
 }
 
+bool CheckRefCount(const std::string &object_id,
```

Comment:
It would be better to accept a `std::unordered_map<std::string, std::pair<size_t, size_t>>` object, so that we can check for multiple objects in one call, just like Python and Java tests.